pkg/logs/input/file/tailer.go

```diff
@@ -210,7 +208,7 @@ func (t *Tailer) startStopTimer() {
 
 // onStop finishes to stop the tailer
 func (t *Tailer) onStop() {
-	log.Info("Closing", t.file.Path, "for tailer key", t.file.GetScanKey())
+	log.Info("Closing", t.file.Path, "for tailer key", t.file.GetScanKey(), "read", t.bytesRead, "bytes and", t.decoder.GetLineCount(), "lines")
```

Comment:
The decoder isn't stopped yet at this point so technically the number of lines decoded may not be final yet when this message is logged.

We could maybe move this message to after the decoder is stopped (and change it to `Closed ...`).